### PR TITLE
Fix Nginx's config

### PR DESCRIPTION
**Pull request by dunglas**
Fixes #308.

### CODE REVIEW

File: docker/nginx/conf.d/default.conf [hunk 1]
```diff
@@ -49,7 +50,4 @@ server {
     location ~ \.php$ {
       return 404;
     }
-
-    error_log /var/log/nginx/app_error.log;
```
Comment (Simperfit) [inline docker/nginx/conf.d/default.conf]: why removing the logs ?

Comment (dunglas) [inline docker/nginx/conf.d/default.conf]: I don't remove them, I keep the default location. It allows to have it both in files and on stdout/stderr thanks to a trick in the official Nginx image.

File: docker-compose.yml [hunk 2]
```diff
@@ -39,6 +39,8 @@ services:
     build:
       context: ./docker/nginx
       dockerfile: ./Dockerfile
+    depends_on:
+      - app
```
Comment (teohhanhui) [inline docker-compose.yml]: This is unnecessary. That's why I removed it.

File: docker/nginx/conf.d/default.conf (hunk 1 above)
Comment (teohhanhui) [inline docker/nginx/conf.d/default.conf]: > It allows to have them both in files and in stdout/stderr thanks to a trick in the official Nginx image.

That's not true. They are symlinked to stdout and stderr respectively.

File: docker/nginx/conf.d/default.conf [hunk 3]
```diff
@@ -1,5 +1,6 @@
 server {
     root /srv/api-platform/web;
+    resolver 127.0.0.11;
```
Comment (teohhanhui) [inline docker/nginx/conf.d/default.conf]: Good catch! I should have actually tried running it!

File: docker/nginx/conf.d/default.conf (hunk 1 above)
Comment (dunglas) [inline docker/nginx/conf.d/default.conf]: It was was I was calling a trick.

**Pull request merged**